README.md

```diff
@@ -47,7 +47,7 @@ Please view the [contributing](#contributing) section below.
 | Development                   | development@rockylinux.org    |
 | Infrastructure                | infrastructure@rockylinux.org |
 | Security                      | security@rockylinux.org       |
-| Web                           | web@rockylinux.org            |
+| Web and Branding              | web@rockylinux.org, brand@rockylinux.com  |
```

Comment:
Please, revert this. There is already a pull request for this. I did not ask you to do this.

Reply:
Okay ! ✌